src/components/pages/InstructorNewsFeed/NewsFeedPostModal.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { Form, Input, Button } from 'antd';
+import axios from 'axios';
+import '../../../styles/InstructorStyles/index.less';
+
+const NewsfeedPostModal = ({ setPostOptions }) => {
+  const [formValue, setformValue] = useState({
+    link: '',
+    description: '',
+    title: '',
+  });
+
+  const handleChange = e => {
+    setformValue({
+      ...formValue,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const token = JSON.parse(localStorage.getItem('okta-token-storage'));
+  const config = {
+    headers: { Authorization: `Bearer ${token.idToken.value}` },
+  };
+
+  const handleSubmit = () => {
+    axios
+      .post('https://coder-heroes-api.herokuapp.com/news', formValue, config)
+      .then(resp => {
+        setPostOptions('newsFeed');
+      })
+      .catch(err => {
+        console.log(err);
+      });
+  };
+
+  return (
+    <div className="newsfeedForm_container">
+      <div className="newsfeedForm_header">
+        <h1>Create New Post</h1>
+        <h2
+          onClick={() => {
+            setPostOptions('newsFeed');
+          }}
+        >
+          x
+        </h2>
+      </div>
+      <Form onFinish={handleSubmit}>
```

Comment:
I've never seen onFinish before! Would you mind explaining why you chose this over onSubmit? Are there certain advantages/disadvantages to each option that you were taking into consideration when making this decision?

Reply:
for the Form for ant design, they use onFinish instead of onSubmit. but it has the same functionality